miden-lib/src/wallets/mod.rs

```diff
@@ -3,11 +3,13 @@ use miden_objects::{
     accounts::{Account, AccountCode, AccountId, AccountStorage, AccountType, AccountVault},
     assembly::ModuleAst,
     crypto::merkle::MerkleStore,
+    utils::string::ToString,
     utils::{format, string::String, vec},
```

Comment:
We can probably combine these two imports into `utils::{format, string::{ToString, String}, vec}`